README.md

```diff
@@ -35,17 +35,19 @@ make unit-test
 # run bdd tests
 make bdd-test
 ```
-##### Running demo locally
-The demo is to play around with APIs using the OpenAPI (Swagger) user interface and JSON.
 
-The demo can be launched by `make run-openapi-demo`
-This target will start Alice and Bob demo agent containers. Once started, OpenAPI user interface for agents can be accessed using below URLs.
+### Running demo locally
+Click [here](docs/demo/openapi_demo_instructions.md) for the detailed instructions to run the OpenAPI demo locally.
```

Comment:
Usually links are not with the text click here.

Reply:
@troyronda [Updated](https://github.com/hyperledger/aries-framework-go/compare/9cc50d4eaedbaa61a0c39767aed3cebc18a7da5a..38d37de61571e17709449b9031e41e4db1d9cc7e#diff-04c6e90faac2675aa89e2176d2eec7d8R40) the line with link in found here.